build.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
-echo "Build Script for eduVPN (and derivatives)"
+
+APPNAME = "eduVPN"
```

Comment:
No, this is not what I meant. This doesn't help at all. You should use the $TARGET when creating the DMG.

Reply:
case "$choice" in
  1 ) TARGET="$APPNAME"; PRODUCT="$APPNAME.app";;
  2 ) TARGET="LetsConnect"; PRODUCT="Let's Connect!.app";;
  * ) echo "Invalid response."; exit 0;;

Reply:
we have put that in TARGET too :)

Reply:
Yes, but that doesn't solve anything. The $APPNAME variable doesn't help. What is needed is that where eduVPN is used as a string in the creation of the DMG, the $TARGET variable is used.

Reply:
Ok Johan , while creating DMG, the TARGET variable in used